guide/C/ch_invest.xml

```diff
@@ -1442,37 +1424,66 @@ Income
           fetch the current prices of your stocks. The file specified
-          <filename class="directory">$HOME/gnucash-filename</filename> will depend on the name and
+          <filename><replaceable>${HOME}/gnucash-filename</replaceable></filename> will depend on the name and
           location of your data file. This can be determined by the name displayed in the top frame
           of the &app; window, before the <quote>-</quote>. The file name can also be found under
           <guimenu>File</guimenu> in the recently opened file list; the first item, numbered 1, is
           the name of the currently open file.
         </para>
 
-        <para>This can be automated by creating a <application>crontab</application> entry. For example, to update
-          your file every Friday evening (16:00) after the relevant exchange markets close (modify
-          the time accordingly for your time zone), you could add the following to your personal
-          <application>crontab</application>:
+        
+      </sect3>
+
+      <sect3 id="invest-stockprice-auto-scheduler">
+        <title>Configuring for Getting Online Quotes Periodically</title>
+        
+        <para>You have to register the <command>gnucash-cli</command> with a scheduler in order to get Online Quotes automatically and periodically.
+          The method depends on your OS. For example, suppose you want to update
+          your file on 16:00 on every Friday.
         </para>
 
-        <para>0 16 * * 5 gnucash --add-price-quotes $HOME/gnucash-filename &gt; /dev/null 2&gt;&amp;1
+        <procedure>
+          <title>Register <command>gnucash-cli</command> with <application>cron</application> on &lin; and &mac;</title>
+          <step><simpara>Run <command>crontab -e</command>.</simpara></step>
+          <step><para>Add the following line to your crontab:
+            <informalexample>
+              <screen>0 16 * * 5 gnucash-cli --quotes get <replaceable>${HOME}/gnucash-filename</replaceable> &gt; /dev/null 2&gt;&amp;1</screen>
+            </informalexample></para>
+          </step>
+        </procedure>
+
+        <procedure>
+          <title>Register <command>gnucash-cli</command> with Task Scheduler on &win;</title>
+          <step><simpara>Select <menuchoice><guimenu>Start</guimenu><guisubmenu>Windows Administrative Tools</guisubmenu><guimenuitem>Task Scheduler</guimenuitem></menuchoice>.</simpara></step>
+          <step><simpara>Select <guimenuitem>Create Task</guimenuitem>.</simpara></step>
+          <step><simpara>Enter the appropirate items on <guilabel>Create Task</guilabel> Window.</simpara></step>
+        </procedure>
+
+        <para>See <ulink url="https://code.gnucash.org/docs/C/gnucash-help/acct-create.html#Online-price-setup">GnuCash Help Manual</ulink> 
+          and <ulink url="https://wiki.gnucash.org/wiki/Online_Quotes#Updating_enabled_Quotes_from_outside_GnuCash">Online Quotes Page on GnuCash Wiki</ulink> for more details.
```

Comment:
We have entities for this urls:
url="&url-docs-build-C;help/acct-create.html#Online-price-setup"
You could use &url-wiki;, but here we should define:
`<!ENTITY url-wiki-OQ "https://wiki.gnucash.org/wiki/Online_Quotes">`

Reply:
56c74d57